packages/leveldb/test.mjs

```diff
@@ -0,0 +1,22 @@
+// node test.mjs
+
+import Level from 'level'
+import pull from 'pull-stream'
+import pullLevel from 'pull-level'
+
+const db = Level('/tmp/level-test')
+
+await db.put('dog', 'tony')
+await db.put('cat', 'nala')
+
+await pull(
+  pullLevel.read(db),
+  pull.map(({ key, value }) => {
+    console.log(key, '=', value)
+    return
+  }),
+  pull.collectAsPromise()
+)
```

Comment:
this comparable code runs perfectly

Reply:
can u try running this in 1 subthread or something like this?

obviously 2 threads will not work but what i want to see is, if the solution is just replacing jest by mocha or whatever.
I appreciate this changes a lot man, thank you!